tools/revng-lift/JumpTargetManager.cpp

```diff
@@ -1812,7 +1812,7 @@ void JumpTargetManager::harvestWithAVI() {
 
     // We're jumping to a single symbol
     bool IsPCStore = TrackedInstructionType::WrittenInPC;
```

Comment:
This line does not make any sense.
I think the following is the proper fix:
```suggestion
    bool IsPCStore = (TIT == TrackedInstructionType::WrittenInPC);
```

Reply:
Thanks for reviewing this! Looks more appropriate, changed it. It fixes the issue and the tests pass successfully.